projects/ngrx.io/content/marketing/resources.json

```diff
@@ -29,6 +29,12 @@
             "url":
               "https://www.youtube.com/playlist?list=PL8OUS498tQP3FFsZzULTGnbyIcILFjHd3",
             "rev": true
+          },
+          "International-Javascript-Conference-London-2019": {
+            "title": "State Management in Angular with NgRx Stores and Effects",
```

Comment:
Just want to denote where the talk took place.

```suggestion
            "title": "State Management in Angular with NgRx Stores and Effects at iJS London 2019",
```

Reply:
Just added the proposed changes. Thank you, nice catch.